<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Login</class>
 <widget class="QMainWindow" name="frm_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2F4F4F;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_usuario">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
    </property>
    <property name="text">
     <string>Usu√°rio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Usuario">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 30px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #00000
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_senha">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Senha">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 30px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #00000
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background: url(:/icon_login/avsIconLogo.png);
	background-repeat: no-repeat; 
    background-position: center; 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>AVS-SOFTWARES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>590</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 16px;
    color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>SOFTWARE E TECNOLOGIA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    font-size: 60px;
    font-weight: bold;
    color: #FFA500;
    padding: 10px;
    border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>BEM-VINDO!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon_TelaLogin.qrc"/>
 </resources>
 <connections/>
</ui>
